### PR TITLE
docs(readme): Add quickstart section

### LINKED ISSUE #36: Obviously state what fat jar is

### LINKED ISSUE #43: Clarification of usage / Quickstart
Hey!

I just used it, great program!
Since I had some troubles starting, I would recommend making a clear "Quickstart" section at the top of the ReadMe.
I would do it if PRs that don't change the Java/Kotlin code are welcome?
Thanks in advance.

### PR DESCRIPTION
**Pull request by FGBxRamel**
This PR adds a quickstart section, mainly for users coming from [MMPA](https://blog.mmpa.info). This should help users which are not very familiar with Java programs.

## Todo

- [x] Make quickstart section
- [x] GUI Mode
- [x] CLI Mode

Closes https://github.com/KosmX/jneedle/issues/43

### CODE REVIEW

**Comment by UltraBlackLinux** [comment]
Okay so I just found this tool and I thought it would be a good idea to set it up just because a few vulnerabilities popped up recently. I have some feedback on your guide, which I think leaves some more info to be desired:
1. What does the tool do when it detects something
2. How do I check if it's working
My setup seemingly ignores jneedle, even though I followed the recommended setup instructions. I think that an indication if it's working is the .jneedle file (directory?) used by  the tool in the home directory. It has not been created so I suspect that something is wrong with my setup.

Edit: I might have found the issue:
> 1. Add the following to the json:
`"+jvmArgs": ["-Ddev.kosmx.jneedle.launchClass={launchClass}"]`, where you replace {launchClass} with the earlier saved class.
> 2. Add the following to the json:
`"mainClass": "dev.kosmx.needle.launchWrapper.ParameterizedWrapper",`

What is not made clear here is that `mainClass` seemingly has to be changed in the modloader configuration, to stop pointing at the modloader and instead point at jneedle. Furthermore, it's even mentioned previously that you could also press "Revert" after getting the modloader entry point, which is all the more confusing if what I'm describing is the correct way to set this up

**Comment by FGBxRamel** [comment]
Thank you for your input.
I will add the 1st point.
The second point is not part of my quickstart. The quickstart is for running either the GUI or CLI version, not integrating it somewhere, as that's not quick and most users don't need it.  
But @KosmX may want to have a look at the second point.

**Comment by UltraBlackLinux** [comment]
I have one more thing to add

 The JSON file documentation is misleading at best. "Adding" that mentioned line to the config file does not simply work (mind the people adding that below the closing bracket too), unless you add it between existing lines. The trailing comma, or lack thereof on the previous line in that file, causes syntax errors that also break jar loading.

Edit: Fixup

**Comment by FGBxRamel** [comment]
Please make separate issues for that, as it has nothing to do with this PR. Thank you.

**Comment by UltraBlackLinux** [comment]
Huh? My second (now only) point is still valid. The (originally) first one seems wrong after more testing. I can't even remember why I included that here.

Why such a passive aggressive response tho

**Comment by FGBxRamel** [comment]
Then I don't quite understand what you mean. I didn't write any "JSON file documentation", neither in this PR nor in the project in general. May you elaborate please?

Sorry if you think that was passive aggressive, wasn't meant like that^^
I really appreciate the input, just don't want to clutter up the PR with unrelated things.

**Comment by UltraBlackLinux** [comment]
I mean `"+jvmArgs": ["-Ddev.kosmx.jneedle.launchClass={launchClass}"], ` and the other, similar line. Do you get the issue with that comma?

**Comment by FGBxRamel** [comment]
Ohhh I see what you mean. Sorry, that confusion is totally on me. 
I didn't actually edit these lines, VS Code just formatted them (I should check my autoformat options...), that's why GitHub shows them as edited.

**Comment by UltraBlackLinux** [comment]
oh no github does not show them as edited, but I meant that this deserves some clearing up since you are trying to make this whole readme a more beginner friendly anyways.
If you are a total beginner, you probably won't know how JSON works, which is why I think it should be pointed out where it's best to paste that line.

**Comment by KosmX** [comment]
I'll prioritize fixing some bugs and rewriting the prismlauncher thingy, but that's a separate issue.  
Probably I'll make a json template that will download the jar itself from a maven server, users can simply `Import Component` that json. And ideally a template selector to my website so users don't have to edit anything.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
+
+### GUI (Graphical) mode
+
+The easiest way to use jNeedle is to use the GUI version.
+Just download the `jneedle-gui-xxx.jar` from the [release page](https://github.com/KosmX/jneedle/releases/latest) and double click on it.
+
+_Help, it won't start!_
+
+There currently is a bug in the GUI version with Java 1.8, which won't let the program start. This is known and will be fixed.  
+If you still want to use it, look at the [CLI mode](#cli-mode).
```
Comment (KosmX) [inline README.md]: Right now there are multiple issues with java 1.8 affecting the scanning. As soon as I can fix those I'll edit this, until then 1.8 is not working at all (or with limited performance)

**Comment by KosmX** [comment]
Approve and run actually does not do anything as you did not modify any code.

File: README.md (hunk 1 above)
Comment (FGBxRamel) [reply README.md]: Will update that

Comment (FGBxRamel) [reply README.md]: Fixed

**Comment by KosmX** [comment]
Thank you!

**Review by KosmX — APPROVED** [review]

**Pull request merged**